CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add hideUnavailableItems at store.config
+  
+### Added
+
```

Comment:
There's already an "Added"  heading above so no need to repeat this one. Sorry if I wasn't clear!
```suggestion
```
I'm going to commit this suggestion, okay?